extensions/amp-story/1.0/amp-story.css

```diff
@@ -380,18 +380,13 @@ amp-story-grid-layer .i-amphtml-embedded-component::after {
   padding: 0 !important;
 }
 
-.i-amphtml-story-grid-template-fill > :not(:first-child) {
-  display: none !important;
-}
-
-.i-amphtml-story-logo {
-  display: none !important;
-  margin: 15px !important;
-}
-
-.i-amphtml-story-grid-template-fill > :first-child {
+.i-amphtml-story-grid-template-fill > * {
+  /**
+   * Publishers need to be able to override the display of an amp-img or
+   * amp-video from their custom CSS, to toggle assets based on media queries.
+   */
+  display: block;
```

Comment:
Sorry, why does this need to be set in general?  Will publishers know how to effectively override this, given that it has a higher specificity?